### DISCUSSION TITLE
Support running without systemd for health checks

### DISCUSSION THREAD

**Discussion by flixman** [containers/podman-compose#1084] [Ideas]
podman relies on systemd to schedule the health checks. In case one wants to run PinP, either systemd needs to be run by the wrapping container, or the checks need to be done manually. How would you guys feel if a flag was provided that would enable systemd-providing features to be replaced by podman-compose? Then we could run the checks as scheduled async tasks.

**Comment by fenchu** [comment]
This would be great, Systemd is not enabled in wsl (windows subsystem for Linux). 
```
$ systemctl list-jobs
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=24.04
DISTRIB_CODENAME=noble
DISTRIB_DESCRIPTION="Ubuntu 24.04.3 LTS"
```
So it means we cant use podman-compose for localhost testing
